lib/mutant/ast/regexp/transformer.rb

```diff
@@ -0,0 +1,187 @@
+# frozen_string_literal: true
+
+module Mutant
+  module AST
+    module Regexp
+      # Regexp bijective mapper
+      #
+      # Transforms parsed regular expression representation from
+      # `Regexp::Expression` instances (provided by `regexp_parser`) into
+      # equivalent representations using `Parser::AST::Node`
+      class Transformer
+        include AbstractType
+
+        REGISTRY = Registry.new
+
+        # Lookup transformer class for regular expression node type
+        #
+        # @param type [Symbol]
+        #
+        # @return [Class<Transformer>]
+        def self.lookup(type)
+          REGISTRY.lookup(type)
+        end
+
+        # Register transformer class as responsible for handling node type
+        #
+        # @param type [Symbol]
+        #
+        # @return [undefined]
```

Comment:
Mutant has stopped to add YARD docs for all private methods. I'm happy to remove them by myself, no need to do it by yourself.

Reply:
That's ok, I'll go ahead and clean them up--I thought you might prefer that but decided I would wait until a flag since I found some of them useful when I was going back through this code.

Reply:
Ok, I think I have resolved this.

Reply:
Going to mark this as resolved unless you find other cases you want to highlight.